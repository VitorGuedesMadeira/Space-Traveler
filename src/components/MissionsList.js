import React from 'react';

const MissionsList = () => (
  <>
    <p>Missions 1</p>
    <p>Missions 2</p>
    <p>Missions 3</p>
    <p>Missions 4</p>
  </>
);

export default MissionsList;
