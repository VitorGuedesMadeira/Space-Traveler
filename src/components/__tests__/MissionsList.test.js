import { render } from '@testing-library/react';
import MissionsList from '../MyProfile';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import React from 'react';

describe('MyProfile component', () => {
  test('Renders MyProfile component', () => {
    const tree = render(
    <React.StrictMode>
      <Provider store={store}>
        <Router>
          <MissionsList />
        </Router>
      </Provider>
    </React.StrictMode>,)
    expect(tree).toMatchSnapshot()
  })
});