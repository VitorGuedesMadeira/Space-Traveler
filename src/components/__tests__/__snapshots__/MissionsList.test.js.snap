// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile component Renders MyProfile component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="my-profile-section"
      >
        <div
          class="rockets-profile"
        >
          <h2>
            Rockets
          </h2>
          <ul>
            <li
              class="no-rocket-reservations"
            >
              No Reservations
            </li>
          </ul>
        </div>
        <div
          class="missions-profile"
        >
          <h2>
            Missions
          </h2>
          <ul>
            <li
              class="no-mission-reservations"
            >
              No Missions
            </li>
          </ul>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="my-profile-section"
    >
      <div
        class="rockets-profile"
      >
        <h2>
          Rockets
        </h2>
        <ul>
          <li
            class="no-rocket-reservations"
          >
            No Reservations
          </li>
        </ul>
      </div>
      <div
        class="missions-profile"
      >
        <h2>
          Missions
        </h2>
        <ul>
          <li
            class="no-mission-reservations"
          >
            No Missions
          </li>
        </ul>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
